docs/api/geo_datasets.csv

```diff
@@ -0,0 +1,17 @@
+Dataset,Type,Source,Size (px),Resolution (m)
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
+`Aster Global DEM`_,Mask,Aster,"3,601x3,601",30
+`Canadian Building Footprints`_,Labels,Generated,,
```

Comment:
Many of these links actually go to the external location specified in the class docstring (e.g., https://github.com/Microsoft/CanadianBuildingFootprints), not to the class section header. I guess the easiest way to prevent this is to add a custom anchor to these sections with a different name? Or maybe a `__` at the end of one of the locations to allow them to be different? Will investigate.

Reply:
The `__` at the end of each link seems to fix the issue. Not sure how to enforce this other than checking each time.